acquia/blt
# Fixes #3386 to add drupal check for deprecated module / theme code.

Fixes #3386 
--------

Changes proposed
---------
(What are you proposing we change? How does this impact end users? Are manual or automatic updates required?)

- adds the mglaman/drupal-check package to composer
- adds the `tests:deprecated:modules` command
- adds the `tests:deprecated:themes` command
- expands `blt validate` command to check modules and themes for deprecated code if `validate.deprecation` values are overridden in blt.yml. 

Steps to replicate the issue
----------
1. add validate.deprecation.modules and validate.deprecation.themes to blt.yml and set to try
2. run blt validate
3. confirm that drupal-check is running for both custom modules and themes.

Additional details
-----------
This is now possible thanks to work being performed in https://github.com/mglaman/drupal-check/issues/68
